engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStoreBenchmark.scala

```diff
@@ -0,0 +1,52 @@
+package cromwell.engine.workflow.lifecycle.execution
```

Comment:
I am super +1 on having stuff like this. However it's not clear to me how I'd run this, when I'd run this, if it's automatically run, etc etc.